Src/BootCamp.Chapter/LogUtility/Logger.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BootCamp.Chapter.LogUtility;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Logger
+    {
+        private const string LogPath = @"..\..\..\Output\Log.txt";
+
+        public Logger()
+        {
+            File.Delete(LogPath);
+        }
+
+        private const int ConsoleLogger = 1;
+        private const int FileLogger = 2;
+        private readonly int LogOption = ConsoleLogger; //Set to either ConsoleLogger or FileLogger
+        private ILogger _logger;
+
+        public void Log(string msg)
+        {
+            switch (LogOption)
```

Comment:
Don't use switch. the whole point was to set a logger based on some option and consistently use it all across the application without the need of an if statement (using polymorphism). This is also inneficient.

Reply:
I have eliminated the switch statement to decide which log option to use.  Instead, I am now using the newly create GetLogOption method.  This is based on if my understanding of how you expect the log option to be chosen is correct or not.